.github/workflows/rust.yml

```
@@ -2,132 +2,104 @@
-      - uses: actions/checkout@v4
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: nightly
-          override: true
-      - uses: Swatinem/rust-cache@v2
-      - name: Generate documentation
-        run: cargo doc --lib --no-deps --all-features --document-private-items
-        env:
-          RUSTFLAGS: ${{ env.RUSTFLAGS }} --cfg docsrs
-          RUSTDOCFLAGS: --cfg docsrs -Dwarnings
+    - uses: actions/checkout@v4
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)